mqtt-sn-2.0/prose/edit/src/images/the-state-diagram.txt

```diff
@@ -4,15 +4,15 @@
  .------------------------------->+  Disconnected  +<----------------------.           
  |                                |                |                       |           
  |                                 '----+----+----'                        |           
- |                                      ^    |                             |           
+ |                                      |    ^                             |           
  |                                      |    |                             |           
  |                                      |    |                             |           
  |                                      |    |                             |           
  |                              CONNECT |    | DISCONNECT                  |           
  |                                      |    |                             |           
  |                                      |    |                             |           
  |                                      |    |                             |           
- |                                      |    v                             |           
+ |                                      v    |                             |           
```

Comment:
I fixed the leftover :+1: to complete the untwisting of the arrow directions versus label meaning problem. Thanks for noticing the twisted logic in the first place.